libcxx/test/std/utilities/variant/variant.variant/variant.assign/conv.pass.cpp

```diff
@@ -35,7 +35,7 @@ int main(int, char**)
   static_assert(!std::is_assignable<std::variant<int, const bool>, decltype("meow")>::value, "");
   static_assert(!std::is_assignable<std::variant<int, const volatile bool>, decltype("meow")>::value, "");
```

Comment:
This line now causes an error in C++26 because

> error: volatile-qualified parameter type 'const volatile bool' is deprecated [-Werror,-Wdeprecated-volatile]

Since `std::variant` no longer treats `const volatile bool` specially, maybe this line could be deleted?

Reply:
Done.